### PR TITLE
04 view all articles

### PR DESCRIPTION
**Pull request by smlbrd**

### CODE REVIEW

File: src/components/ArticleCard.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import timestampToDate from '../utils/timestampToDate';
+
+function ArticleCard({ article_img_url, created_at, title, topic, cardId }) {
+  const date = String(timestampToDate(created_at));
```
Comment (nc-mezz-davies) [inline src/components/ArticleCard.jsx:4]: Nice 👍

File: src/api/api.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
```
Comment (nc-mezz-davies) [inline src/api/api.js:1]: really good you have done this separation of concerns and used the baseURL 👍

File: src/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+function Header() {
```
Comment (nc-mezz-davies) [inline src/components/Header.jsx:1]: If you were to add any other content to this component you should use the header html element as a wrapper

File: src/components/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,3 @@
+function Footer() {}
+
+export default Footer;
```
Comment (nc-mezz-davies) [inline src/components/Footer.jsx:3]: Make sure to use the semantic footer html tag when you add content to this component

**Review by nc-mezz-davies — COMMENTED** [review]
This is looking good, but a few points to consider:

- There should be some visual feedback to the user whilst data is loading (think slow network!)
- The article cards are rendered in a ul element but they are divs so the ul has no list items - not great for accessibility/screen-readers
- currently your site is not responsive and I have to use horizontal scroll. Complete styling is not required but responsive design should be considered.

File: src/components/Header.jsx (hunk 3 above)
Comment (smlbrd) [reply src/components/Header.jsx:1]: Updated! :saluting_face:

File: src/components/Footer.jsx (hunk 4 above)
Comment (smlbrd) [reply src/components/Footer.jsx:3]: Updated! :saluting_face:

**Pull request merged**